### ISSUE TITLE
Custom filters not working with include:filter

### ISSUE THREAD

**Issue by dcposch** [pugjs/pug#2440]
### 1. Make a custom filter

```js
pug.filters.myfilter = (text, options) => {
  console.log('TEXT', text)
  console.log('OPTIONS', options)
  return text.toLowerCase()
}
```

### 2. Use it in a block. Works fine.
```
block body
  :myfilter
    TESTING 123
```

...if you run that, the page reads 'testing 123', and here's the console:
```
TEXT: TESTING 123
OPTIONS: {'filename': '[...]/test.pug'}
```

### 3. Use it together with `include`. Doesn't work.
```
block body
  include:myfilter myfile.txt
```

...if you run that, the page just contains the full path to `myfile.txt`, not the contents! Here's the console:
```
TEXT: [...]/myfile.txt
OPTIONS: {'filename': '[...]/myfile.txt'}
```

When you use `include:myfilter`, Pug should read the file and pass the contents to the filter, not pass the filename, right?

**Comment by ayroblu** [comment]
Would also like to add a new test case to handle this, but I'm not familiar with how you generate them?

**Comment by TimothyGu** [comment]
I opened a ticket in pugjs/pug-filters#18 for the addition of a test case.

**Comment by ForbesLindesay** [comment]
Re-opening until someone has confirmed that the update still works with builtin filters.

**Comment by TimothyGu** [comment]
Really fixed in pug-filters@1.2.3/pug@2.0.0-beta4.
